src/richie/apps/courses/templates/courses/cms/fragment_organization_glimpse.html

```diff
@@ -30,8 +30,23 @@
             {% endblockplugin %}
         </div>
         <div class="organization-{{ organization_variant }}__content">
-            <h{{ header_level|default:3 }} class="organization-{{ organization_variant }}__title" property="name">{{ organization_page.get_title }}</<h{{ header_level|default:3 }}>
-            <div class="organization-{{ organization_variant }}__excerpt" property="description">{% get_placeholder_plugins "description" organization_page as plugins or %}{% trans "Description" %}{% endget_placeholder_plugins %}{% blockplugin plugins.0 %}{{ instance.body|truncatewords_html:40|safe }}{% endblockplugin %}</div>
+            <h{{ header_level|default:3 }} class="organization-{{ organization_variant }}__title" property="name">{{ organization_page.get_title }}</h{{ header_level|default:3 }}>
+            <div class="organization-{{ organization_variant }}__excerpt" property="description">
+                {% get_placeholder_plugins "excerpt" organization_page as plugins or %}
+                    <p>{% trans "Excerpt" %}</p>
+                {% endget_placeholder_plugins %}
```

Comment:
When there is no plugin in this placeholder, `Excerpt` is displayed.

<img width="791" alt="image" src="https://user-images.githubusercontent.com/9265241/158413603-b301c6df-dd93-49c1-809a-f0479b9b0c51.png">

Reply:
Yeah i reproduced for excerpt the same behavior than with description in this template variant but i can remove it if it is not wanted

Reply:
Ok i've done it as you said.